tests/pyspark-notebook/test_spark.py

```diff
@@ -16,3 +16,21 @@ def test_spark_shell(container: TrackedContainer) -> None:
     )
 
     assert "res0: Int = 2" in logs, "spark-shell does not work"
+
+
+def test_pandas_version(container: TrackedContainer) -> None:
+    """Checking if pandas is installed and its version is 1.5.3
+    This is only needed for Spark before 4.0 see #1924"""
+    command = [
+        "start.sh",
+        "bash",
+        "-c",
+        "python -c 'import pandas; print(pandas.__version__)'",
```

Comment:
Actually, we have a simple way of running python scripts inside a container.
Take a look here: https://jupyter-docker-stacks.readthedocs.io/en/latest/contributing/tests.html#unit-tests